ERROR:root:ZeroDivisionError
Traceback (most recent call last):
  File "/tmp/ipykernel_4516/769244530.py", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero
ERROR:root:ZeroDivisionError
Traceback (most recent call last):
  File "/tmp/ipykernel_4516/769244530.py", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero
ERROR:root:ZeroDivisionError
Traceback (most recent call last):
  File "/tmp/ipykernel_4516/1171463990.py", line 2, in <module>
    1/0
    ~^~
ZeroDivisionError: division by zero
ERROR:root:ZeroDivisionError
Traceback (most recent call last):
  File "/tmp/ipykernel_4516/8376821.py", line 2, in <module>
    1 / 0
    ~~^~~
ZeroDivisionError: division by zero
ERROR:Comm:No such comm target registered: jupyter.widget.control
ERROR:Comm:No such comm target registered: jupyter.widget.control
